test/t_web/channels/feed_channel_live_test.exs

```diff
@@ -283,44 +283,46 @@ defmodule TWeb.FeedChanneLiveTest do
       we = onboarded_user(gender: "M", accept_genders: ["F", "N"])
       joined_mate(%{mate: we})
 
-      p1 = onboarded_user(gender: "F", accept_genders: ["F", "N", "M"])
+      %{id: user_id} = p1 = onboarded_user(gender: "F", accept_genders: ["F", "N", "M"])
       joined_mate(%{mate: p1})
 
-      assert_push("activated_profile", _profile)
+      assert_push "activated_profile", %{"profile" => %{user_id: ^user_id}}
```

Comment:
These tests used to pass when they shouldn't have when run concurrently with tests in `feeds_live_tests.exs`, so I added a pattern with user id to match.

Reply:
In `feeds_live_tests.exs` there were sessions being activated with users without stories, which were broadcasted to out channel here making the channel crash.